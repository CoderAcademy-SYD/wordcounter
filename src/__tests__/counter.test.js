import React from 'react';
import renderer from 'react-test-renderer';
import Counter from '../Counter.js';

describe("Counter", () => {
  it("Displays the count and the label", () => {
    const counter = renderer.create(
      <Counter legend="Count" count={34}/>
    );
    expect(counter.toJSON()).toMatchSnapshot();
  });
});
