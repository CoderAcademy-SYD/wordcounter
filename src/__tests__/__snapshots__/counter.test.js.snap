// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter Displays the count and the label 1`] = `
<p
  className="mb2"
>
  <label
    htmlFor="count"
  >
    Word Count:
  </label>
  <output
    id="count"
  >
    34
  </output>
</p>
`;
